recipes/tclap/1.2.3/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, tools
+
+class TclapConan(ConanFile):
+    name = "tclap"
+    version = "1.2.3"
+    license = "MIT License"
+    homepage = "http://github.com/xguerin/tclap"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Templatized Command Line Argument Parser"
+    topics = ("c++", "commandline parser")
+    no_copy_source = True
+    exports_sources = ["include/*"]
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def package(self):
+        src_folder = self.name + "-" + self.version
+        self.copy("*.h", "include", src_folder + "/include")
```

Comment:
https://github.com/xguerin/tclap/blob/6689a6109700390d61c44298181da7e73fab7dae/tclap.pc.in#L4

```suggestion
        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
        self.copy(pattern="*", src=os.path.join(self._source_subfolder, "include"), dst="include", keep_path=True)

    def package_info(self):
        self.cpp_info.names["pkg_config"] = "tclap"
```